app/update/settings.py

```diff
@@ -46,22 +45,42 @@ def tinypilot_repo_branch(self, value):
         self._data['tinypilot_repo_branch'] = value
 
 
-def load(settings_file):
-    """Create a settings instance to manage the TinyPilot settings file.
+def load():
+    """Retrieves the current TinyPilot update settings
 
     Parses the contents of settings file and generates a settings object that
     represents the values in the settings file.
 
     Args:
-        settings_file: File containing TinyPilot settings in YAML format.
+        None
 
     Returns:
-        An object containing TinyPilot settings.
+        A Settings instance of the current update settings.
     """
+    try:
+        with open(_settings_file_path()) as settings_file:
+            return _from_file(settings_file)
+    except FileNotFoundError:
+        return Settings(data=None)
+
+
+def save(settings):
+    """Saves a Settings object to the settings file."""
+    with open(_settings_file_path(), 'w') as settings_file:
+        _to_file(settings, settings_file)
+
+
+def _from_file(settings_file):
     return Settings(yaml.safe_load(settings_file))
 
 
-def save(settings, settings_file):
-    """Writes the current settings to the settings file."""
+def _to_file(settings, settings_file):
+    """Writes a Settings object to a file."""
     if settings.as_dict():
         yaml.safe_dump(settings.as_dict(), settings_file)
+
+
+# This could be a constant, but we're doing it as a function so that we can mock
+# out the os.path.expanduser call for unit testing.
+def _settings_file_path():
+    return os.path.expanduser('~/settings.yml')
```

Comment:
To avoid making this a function, I think we can still mock a constant using:
```python
_SETTINGS_FILE_PATH = os.path.expanduser('~/settings.yml')
...
mock.patch('update.settings._SETTINGS_FILE_PATH', fake_settings_file_path).start()
```